provisioning_templates/provision/kickstart_default.erb

```diff
@@ -87,7 +87,12 @@ authselect --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downca
 <% else -%>
 authconfig --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downcase || 'sha256' %> --kickstart
 <% end -%>
+<% if (is_fedora && os_major < 16) || (rhel_compatible && os_major < 8) -%>
```

Comment:
Can you extract this into helper variable `use_chrony` please? It's three places and it also enables us to provide host param with the same name:

```ruby
use_chrony = (is_fedora && os_major < 16) || (rhel_compatible && os_major < 8) || ! host_param('use-ntpd')
```

I want to allow users to use ntpd if they need to.